includes/class-sensei-question.php

```diff
@@ -37,11 +37,74 @@ public function __construct() {
 			add_filter( 'request', array( $this, 'filter_actions' ) );
 
 			add_action( 'save_post_question', array( $this, 'save_question' ), 10, 1 );
+
+			// Add custom navigation.
+			add_action( 'in_admin_header', [ $this, 'add_custom_navigation' ] );
+			add_filter( 'submenu_file', [ $this, 'highlight_menu_item' ] );
 		}
 
 		add_action( 'sensei_question_initial_publish', [ $this, 'log_initial_publish_event' ] );
 	}
 
+	/**
+	 * Add custom navigation to the admin pages.
+	 *
+	 * @since 4.0.0
+	 * @access private
+	 */
+	public function add_custom_navigation() {
+		$screen = get_current_screen();
+		if ( ! $screen ) {
+			return;
+		}
+		if ( in_array( $screen->id, [ 'edit-question', 'edit-question-category' ], true ) ) {
+			$this->display_question_navigation( $screen );
+		}
+	}
+
+	/**
+	 * Highlight the menu item for the question pages.
+	 *
+	 * @param string | null $submenu_file The submenu file points to the certain item of the submenu.
+	 *
+	 * @return string | null
```

Comment:
Small thing, but the `@param`s and `@return` are usually the last thing, so below `@since` and `@access`.

Reply:
Thanks, here is the change: https://github.com/Automattic/sensei/pull/4692/commits/63d63f261ce0b2524ae3bb395f4b911e3bea71ba#diff-3486c06ef1c37a423c0b733774e415315237355acdc2c495352576daec34978dR71-R73

I've also removed | null as an option, since condtiaional parameters are not allowed in 7.0